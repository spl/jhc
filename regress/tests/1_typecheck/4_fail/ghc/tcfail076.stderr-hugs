ERROR "tcfail076.hs":18 - Inferred type is not general enough
*** Expression    : \cont' -> cont a
*** Expected type : (_29 -> _28 a) -> _28 a
*** Inferred type : (_29 -> _28 _30) -> _28 _30
