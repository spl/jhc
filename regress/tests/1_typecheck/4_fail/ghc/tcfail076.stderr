
tcfail076.hs:18:82:
    Couldn't match type `res' with `res1'
      `res' is a rigid type variable bound by
            a type expected by the context: (a -> m res) -> m res
            at tcfail076.hs:18:28
      `res1' is a rigid type variable bound by
             a type expected by the context: (b -> m res1) -> m res1
             at tcfail076.hs:18:64
    Expected type: m res1
      Actual type: m res
    In the return type of a call of `cont'
    In the expression: cont a
